cypress/features/visual/alert.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Alert component
+  I want to change Alert component properties
+
+  Background: Open Alert component page
+    Given I open "Alert" component page with button
+
+  @applitools
+  Scenario Outline: Change Alert component title to <title>
+    When I set title to "<title>"
+      And I open component preview
+    Then Element displays correctly
```

Comment:
this step is not checking title, just the correct visibility of inner content of iFrame, so we need to remove from here this step `When I set title to "<title>"` and `Examples`